tests/test_shape.py

```diff
@@ -0,0 +1,103 @@
+                               1.39440954, 4.17895194e-07, 1.90942151e-10,
+                               1.73438394, 8.79363477e-01, -5.82683384e-01,
+                               1.73533678, 8.79923284e-01, -5.83617747e-01,
+                               1.73533118, 8.79921675e-01, -5.83607316e-01,
+                               0., 0., 0.,
+                               0., 0., 0.,
+                               0., 0., 0.,
+                               0., 0., 0.,
+                               0., 0., 0.,
+                               0., 0., 0.,
+                               0., 0., 0.,
+                               0., 0., 0.))
+    methylo_usrcat = np.array((1.57888687, 0.20933244, 0.06888204,
+                               1.70717049, 0.76747602, -0.42800424,
+                               2.63272667, 1.96170557, -1.70290744,
+                               2.10210729, 1.56102777, 0.13983713,
+                               1.57888687, 0.20933244, 0.06888204,
+                               1.70717049, 0.76747602, -0.42800424,
+                               2.63272667, 1.96170557, -1.70290744,
+                               2.10210729, 1.56102777, 0.13983713,
+                               1.42734802, 0.08347309, -0.00348717,
+                               1.74170351, 0.88704103, -0.5897935,
+                               3.07151437, 0.94091368, -0.26844379,
+                               1.7362355,  0.88416433, -0.58055663,
+                               0., 0., 0.,
+                               0., 0., 0.,
+                               0., 0., 0.,
+                               0., 0., 0.,
+                               0., 0., 0.,
+                               0., 0., 0.,
+                               0., 0., 0.,
+                               0., 0., 0.))
+
+    assert_array_almost_equal(usr_cat(benzene), benzene_usrcat)
+    assert_array_almost_equal(usr_cat(methylo), methylo_usrcat)
+
+
+def test_electroshape():
+    benzene_usr = np.array((1.39440954, 6.46448112e-04, 5.75838378e-04,
+                            1.73533678, 9.38042283e-01, -8.35685432e-01,
+                            1.73533118, 9.38041389e-01, -8.35680425e-01,
+                            1.48386667, 4.78803348e-01, -2.95281451e-01,
+                            1.48386667, 4.78803348e-01, -2.95281451e-01))
+    methylo_usr = np.array((1.58802426, 0.46545091, 0.42549801,
+                            2.66513181, 1.41204774, -1.21510851,
+                            2.10947442, 1.25946617, 0.56637883,
+                            1.97007351, 0.78387688, -0.65092859,
+                            1.90027715, 0.85848847, -0.74442503))
+
+    assert_array_almost_equal(electroshape(benzene), benzene_usr)
+    assert_array_almost_equal(electroshape(methylo), methylo_usr)
+
+
+def test_similarity():
+    similarity_usr = 0.68080239527011543
+    similarity_usrcat = 0.6815992248592293
```

Comment:
Please double-check the USRCAT.
Reference values indicate there should be more difference - http://usr.marseille.inserm.fr/iview/?591ac6825807f74598585cdc see hit 7.)